source/documentors/references/quick_reference_rst.rst

```diff
@@ -172,6 +221,111 @@ Use the following code:
 
 .. seealso:: To see alternative ways of defining tables, visit the `RST documentation about this topic <https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/#tables>`_.
 
+Add a Substitution
+******************
+
+In RST, a *substitution* serves as a variable which you can set a value for once, then use repeatedly. This is useful for words or phrases that are used often, as it enables you to edit the value once and change it everywhere.
+
+You also need to use substitutions for inline images, as explained below.
+
+Substitutions are all kept in the source/substitutions.txt file in the documentation project on GitHub.
+
+Copy the format for the substitution as needed.
+
+.. code-block:: RST
+
+  .. |variable name| replace:: value
+
+  .. |Platform name| replace:: Open edX
+
+You then add the *variable name* inline in the topic.
+
+.. code-block:: RST
+
+  A line of text with an |variable name| inserted.
+
+
+Add a Sidebar
+*************
+
+You can add any content in a sidebar. Open edX uses sidebars for image thumbnails, videos, and other notes.
+
+The sidebar must come directly after a heading.
+
+Copy this codeblock to add a new sidebar topic.
+
+.. code-block:: RST
+
+  .. sidebar:: Sample Sidebar
+
+    Any content, typically an image, video, or note.
+
+Add an Image to a Topic
+***********************
+
+You can add an on its own separate line, inline, or in a sidebar.
+
+You can also add an image directly, or add a thumbnail of an larger image, which when clicked on will open the full image.
+
+You must save images in the ``source/_images`` directory before adding a reference to it in a topic.
+
+Add an Image on its Own Line
+============================
+
+Copy this codeblock to an image on its own line.
+
+.. code-block:: RST
+
+  Line of content, followed by a line with an image.
+
+  .. image:: /_images/image-file-name
+
+  Or, a line of content, followed by a clickable thumbnail of a large image.
+
+  .. thumbnail:: /_images/image-file-name
+
+Add an Image Inline
+===================
+
+To add an image inline, you must first create a substitution for the image in the substitutions.txt file.
+
+Copy the format for the substitution as needed.
+
+.. code-block:: RST
+
+  .. |variable name| image:: /_images/image-file-name
+
+You then add the *variable name* inline in the topic.
+
+.. code-block:: RST
+
+  A line of text with an |variable name| inserted.
+
+Add an Thumbnail in a Sidebar
+=============================
+
+You can add a thumbnail in a sidebar, a common practice for How-to topics.
+
+The sidebar must come directly after a heading.
+
+Copy this codeblock to add a new sidebar with a thumbnail.
+
+.. code-block:: RST
+
+  .. sidebar:: Sample Sidebar with a thumbnail
+
+    .. thumbnail:: _images/image-file-name
+
+Use Inline Formatting
```

Comment:
This section is identical to the above https://docsopenedxorg--613.org.readthedocs.build/en/613/documentors/references/quick_reference_rst.html#inline-markup section, so this can be removed.